census/keras/trainer/task.py

```diff
@@ -52,16 +53,20 @@ def __init__(self,
 
   def on_epoch_begin(self, epoch, logs={}):
     if epoch > 0 and epoch % self.eval_frequency == 0:
-      checkpoints = glob.glob(os.path.join(self.job_dir, 'checkpoint.*'))
-      checkpoints.sort()
-      census_model = load_model(checkpoints[-1])
-      census_model = model.compile_model(census_model, self.learning_rate)
-      loss, acc = census_model.evaluate_generator(
-          model.generator_input(self.eval_files, chunk_size=CHUNK_SIZE),
-          steps=self.steps)
-      print '\nEvaluation epoch[{}] metrics[{:.2f}, {:.2f}] {}'.format(
-          epoch, loss, acc, census_model.metrics_names)
 
+      checkpoints = glob.glob(path_join_for_non_gcs(self.job_dir, 'checkpoint.*'))
+      if len(checkpoints) > 0:
+        checkpoints.sort()
+        census_model = load_model(checkpoints[-1])
+        census_model = model.compile_model(census_model, self.learning_rate)
+        loss, acc = census_model.evaluate_generator(
+            model.generator_input(self.eval_files, chunk_size=CHUNK_SIZE),
+            steps=self.steps)
+        print '\nEvaluation epoch[{}] metrics[{:.2f}, {:.2f}] {}'.format(
+            epoch, loss, acc, census_model.metrics_names)
+        maybe_copy_file_to_gcs(self.job_dir, checkpoints[-1])
```

Comment:
change the name of maybe_copy_file... to copy_file_to...

Reply:
maybe_* seems bit odd in the name function name :) Ok lets leave it that way.

Reply:
+1

Reply:
changed it to remove the `maybe` bit since the more explicit conditionals make it kind of superfluous.